packages/suite/src/views/wallet/coinmarket/common/CoinmarketSelectedOffer/CoinmarketSelectedOfferVerifyOptions.tsx

```diff
@@ -53,104 +54,120 @@ const AccountType = styled.span`
     padding-left: 5px;
 `;
 
+const SelectWrapper = styled.div`
+    position: relative;
+    z-index: ${zIndices.navigationBar};
```

Comment:
I'm not sure about this z-index. Does it have anything common with navigationBar?

Reply:
No, it doesn't. Should I rather use a plain number?

Reply:
In this case yes. Thanks 🙏